### DISCUSSION TITLE
2010/12/03/%E6%96%B0%E6%B5%AA%E5%8D%9A%E5%AE%A2%E5%88%A0%E9%99%A4%E7%9A%84%E6%96%87%E7%AB%A0

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#303] [Announcements]
# 2010/12/03/%E6%96%B0%E6%B5%AA%E5%8D%9A%E5%AE%A2%E5%88%A0%E9%99%A4%E7%9A%84%E6%96%87%E7%AB%A0

把博客备份到新浪时，发现少了一些。今天仔细察看了一下，新浪删除了我的一百多篇文章，理由是有敏感内容。新浪给出的列表居然全都是乱码的，我也懒得去查具体是哪些篇了。不过从标题的英文中可以看出，很多文章是纯粹技术的，居然也算敏感内容。由此也可知，新浪根本没有人工审核，设置一些敏感词，然后凡是沾边的，都直接删除了事。

https://blog.qizhen.xyz/2010/12/03/%E6%96%B0%E6%B5%AA%E5%8D%9A%E5%AE%A2%E5%88%A0%E9%99%A4%E7%9A%84%E6%96%87%E7%AB%A0

**Comment by ruanqizhen** [comment]
之前博客下的留言：

mangguopai
2010年12月3日 21:09 编辑
电话联系Sina

回复
